### PR TITLE
Fixing readme template and preview for reports

### PR DESCRIPTION
**Pull request by Red-GV**
This makes each configuration into a directory and copies the template into them. For that scenario (write or read), those graphs will be displayed. However, the other one will not be displayed.

### CODE REVIEW

**Review by periklis — CHANGES_REQUESTED** [review]
The report dir should provide a single report for all scenarios or at least a table of contents linking the subfolder reports.

**Comment by Red-GV** [comment]
Went in favor of having a readme table of contents for linking within the various subfolders. The dynamic part of the multi-configurations is more difficult to converge into one report.

File: benchmarks/benchmarks_suite_test.go [hunk 1]
```diff
@@ -87,12 +92,67 @@ func init() {
...
+
+	title := "# Benchmark Report\n\n" +
+						"This document contains baseline benchmark results for Loki under synthetic load.\n\n"
+	tableOfContents := "## Table of Contents\n\n" +
+										 "- [Benchmark Profile](#benchmark-profile)\n"
+	profileSection := "---\n\n" +
+										"## Benchmark Profile\n\n" +
+										"Generated using profile:\n" +
+										"[embedmd]:# (../../config/{{TARGET_ENV}}.yaml)"
+
```
Comment (periklis) [inline benchmarks/benchmarks_suite_test.go:136]: Indentation broken here?

**Review by periklis — APPROVED** [review]

**Comment by periklis** [comment]
@Red-GV Could you provide also a fix for CircleCI archiving the reports:
```
Unable to save test results from /home/circleci/loki-benchmarks/reports
Error path is not valid "/home/circleci/loki-benchmarks/reports": error accessing path "/home/circleci/loki-benchmarks/reports/2021-07-13-13-02-33/Read-aggregate-203-queries-per-second": open /home/circleci/loki-benchmarks/reports/2021-07-13-13-02-33/Read-aggregate-203-queries-per-second: permission denied
Found no test results, skipping
```

**Pull request merged**